core/src/main/kotlin/org/evomaster/core/search/Individual.kt

```diff
@@ -15,12 +16,28 @@ import org.evomaster.core.search.tracer.TrackOperator
  * to a RESTful API, SQL operations on a database or WireMock setup)
  *
  */
-abstract class Individual(trackOperator: TrackOperator? = null, index : Int = DEFAULT_INDEX) : TraceableElement(trackOperator, index){
+abstract class Individual(override var trackOperator: TrackOperator? = null,
+                          override var index: Int = Traceable.DEFAULT_INDEX,
+                          children: List<out StructuralElement>
+) : Traceable, StructuralElement(children){
+
+    override var evaluatedResult: EvaluatedMutation? = null
+
+    override var tracking: TrackingHistory<out Traceable>? = null
 
```

Comment:
`evaluatedResult` and `tracking`? why here? we do have already a class for `EvaluatedIndividual` . if really needed here, add comments to explain it

Reply:
what does it mean `do not recommend`? are we tracking in both places? what would be the case of `evaluatedIndividual` not been tracked, but `individual` is?

Reply:
or are we not tracking in `evaluatedIndividual`?